.github/workflows/main.yml

```diff
@@ -70,14 +71,8 @@ jobs:
           PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
 
 
-      - name: Checking bundle size of visual testing app
-        run: yarn bundlesize
-        # env:
-        # NOTE: it seems that currently bundlesize server is having problems
-        # resulting in status code 500. Because of that, the bundlesize script
```

Comment:
The issue is that iltorb@2.4.5 (a dependency of brotli-size@0.1.0, which is used by bundlesize) doesn't support Node.js 24 because it uses deprecated V8 APIs.

I'm trying to find a alternate solution to resolve this

Reply:
Unfortunately this new github action also seems to use same `iltorb`. will keep digging

Reply:
Yes. I think we can swap.